sdk/spring/azure-spring-boot/src/main/java/com/azure/spring/autoconfigure/b2c/AADB2CAutoConfiguration.java

```diff
@@ -37,26 +31,19 @@
  * </p>
  */
 @Configuration
-@ConditionalOnWebApplication
-@ConditionalOnResource(resources = "classpath:aadb2c.enable.config")
-@ConditionalOnProperty(
-    prefix = AADB2CProperties.PREFIX,
-    value = {
-        "client-id",
-        "client-secret"
-    }
-)
+@Conditional({ AADB2CConfiguration.AnyCondition.class})
+@ConditionalOnMissingClass({"org.springframework.security.oauth2.server.resource.BearerTokenAuthenticationToken"})
 @EnableConfigurationProperties(AADB2CProperties.class)
+@Import(AADB2CConfiguration.class)
```

Comment:
Addd more doc here to describe when will the auto configuration class be called?

Reply:
@moarychan please add more doc here.